### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by daphneherlambda**
This PR updates readme with new specifications and commands

### CODE REVIEW

**Review by Juan-M-V — APPROVED** [review]

**Review by dafifynn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,28 +30,19 @@ In this repo the fuzzers have the following structure:
...
+### 1. Create the docker container 
+To create and run the docker container ready to run the fuzzers use the command:
+`make docker-up`
 
-* run the command `docker run -it --rm --cap-add=SYS_PTRACE --security-opt seccomp=unconfined ready-to-fuzz-image bash` to run the container from the image previously created.
 
 ### 2. Run a fuzzer 
 
-Depending on the fuzzer use the following commands:
+Each fuzzer has its own Readme with specific instructions to run each of them.
```
Comment (Loco1982) [inline README.md]:
```suggestion
Each fuzzer has its own README with specific instructions to run each of them.
```

File: README.md [hunk 2]
```diff
@@ -30,28 +30,19 @@ In this repo the fuzzers have the following structure:
 
 ## User Guide
 
-### 1. create the docker container 
-* Use the command  `docker build -t ready-to-fuzz-image .` inside this project to create a docker image named **ready-to-fuzz-image** from the dockerfile in the project.
+### 1. Create the docker container 
+To create and run the docker container ready to run the fuzzers use the command:
```
Comment (Loco1982) [inline README.md]: This phrase is confusing. I think it should be

```suggestion
To create and run the docker container with the fuzzers use the command:
```

File: README.md (hunk 1 above)
Comment (Loco1982) [reply README.md]: Also I think it would be better to add links to every fuzzer folder and its README

**Review by Loco1982 — CHANGES_REQUESTED** [review]
Added some suggestions

File: README.md (hunk 1 above)
Comment (daphneherlambda) [reply README.md]: done 👍

File: README.md (hunk 2 above)
Comment (daphneherlambda) [reply README.md]: done 👍

**Review by Loco1982 — APPROVED** [review]
LGTM

**Review by Juan-M-V — APPROVED** [review]

**Pull request merged**